types/ember-osf-web/styles.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*/styles' {
```

Comment:
Won't this interfere with any module named `styles`?

Reply:
Probably